unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/NavMap.prefab

```diff
@@ -3217,7 +3217,7 @@ MonoBehaviour:
   m_text: Scene Title
   m_isRightToLeft: 0
   m_fontAsset: {fileID: 11400000, guid: 26b03903800224f718b64e9afbc91b61, type: 2}
-  m_sharedMaterial: {fileID: 2100000, guid: cbe62851a9f374bb9903a58154b47a77, type: 2}
+  m_sharedMaterial: {fileID: 0}
```

Comment:
Is it ok to drop these materials?

Reply:
I just checked again the prefab, I made minor changes on it:
* removed an unused property from the NavmapView monobehaviour class attached to the root gameobject
* Replaced the ScrollRect native monobehaviour for a new one I made that inherits from that one and overrides one of its methods for the WebGL trackpad scrolling bug fix.

No materials were touched, sometimes when moving things around or changing children position in the hierarchy, unity shows that kind of changes in the YAML, now sure why.